data/json/overmap_locations.json

```diff
@@ -1,62 +1,84 @@
 [
-    {
-        "type": "overmap_location",
-        "id": "field",
-        "terrains": [ "field" ]
-    },{
-        "type": "overmap_location",
-        "id": "forest",
-        "terrains": [ "forest", "forest_thick" ]
-    },{
-        "type": "overmap_location",
-        "id": "swamp",
-        "terrains": [ "forest_water" ]
-    },{
-        "type": "overmap_location",
-        "id": "land",
-        "terrains": [ "forest", "forest_thick", "forest_water", "field" ]
-    },{
-        "type": "overmap_location",
-        "id": "water",
-        "terrains": [
-            "river",
-            "river_center",
-            "river_c_not_ne",
-            "river_c_not_nw",
-            "river_c_not_se",
-            "river_c_not_sw",
-            "river_ne",
-            "river_se",
-            "river_sw",
-            "river_nw"
-        ]
-    },{
-        "type": "overmap_location",
-        "id": "wilderness",
-        "terrains": [ "forest", "forest_thick", "field" ]
-    },{
-        "type": "overmap_location",
-        "id": "subterranean",
-        "terrains": [
-            "cavern",
-            "empty_rock",
-            "hellmouth",
-            "rift",
-            "rock",
-            "slimepit",
-            "sewer",
-            "subway"
-        ]
-    },{
-        "type": "overmap_location",
-        "id": "road",
-        "terrains": [
-            "hiway_ns",
-            "hiway_ew",
-            "ranch_camp_77",
-            "necropolis_a_11",
-            "road",
-            "road_nesw_manhole"
-        ]
-    }
+  {
+    "type": "overmap_location",
+    "id": "field",
+    "terrains": [ "field" ]
+  },
+  {
+    "type": "overmap_location",
+    "id": "forest",
+    "terrains": [ "forest", "forest_thick" ]
+  },
+  {
+    "type": "overmap_location",
+    "id": "swamp",
+    "terrains": [ "forest_water" ]
+  },
+  {
+    "type": "overmap_location",
+    "id": "land",
+    "terrains": [ "forest", "forest_thick", "forest_water", "field" ]
+  },
+  {
+    "type": "overmap_location",
+    "id": "water",
+    "terrains": [
+      "river",
+      "river_center",
+      "river_c_not_ne",
+      "river_c_not_nw",
+      "river_c_not_se",
+      "river_c_not_sw",
+      "river_ne",
+      "river_se",
+      "river_sw",
+      "river_nw"
+    ]
+  },
+  {
+    "type": "overmap_location",
+    "id": "wilderness",
+    "terrains": [ "forest", "forest_thick", "field" ]
+  },
+  {
+    "type": "overmap_location",
+    "id": "subterranean",
+    "terrains": [ "cavern", "empty_rock", "hellmouth", "rift", "rock", "slimepit", "sewer", "subway" ]
+  },
+  {
+    "type": "overmap_location",
+    "id": "road",
+    "terrains": [ "hiway_ns", "hiway_ew", "ranch_camp_77", "necropolis_a_11", "road", "road_nesw_manhole" ]
+  },
+  {
+    "type": "overmap_location",
+    "id": "need_none",
```

Comment:
Since these are locations, would be better to name them `source_of_...`. In this case it would be something like `source_of_random_loot`.